### ISSUE TITLE
Training VOCA question

### ISSUE THREAD

**Issue by Enin** [TimoBolkart/voca#80]
Hello from Korea.

I want to do voca training session for get my own model. first I download vocaset: Audio, Sentences text, Unposed Cleaned data - Each of subject data are not downloaded completely - and question is where I need to put this datasets and what is input and output data. (Guess audio wav file is input and ply mesh data of each subject is output?)

Am I just put these data only and run 'run_training.py' script? Additionally if i want to train my own datasets, it's same as vocaset? (of course I will change 'config_parser.py' script for new dataset) 

Sorry for poor English and thank you for your answer.

**Comment by mahdi943** [comment]
> Hello from Korea.
> 
> I want to do voca training session for get my own model. first I download vocaset: Audio, Sentences text, Unposed Cleaned data - Each of subject data are not downloaded completely - and question is where I need to put this datasets and what is input and output data. (Guess audio wav file is input and ply mesh data of each subject is output?)
> 
> Am I just put these data only and run 'run_training.py' script? Additionally if i want to train my own datasets, it's same as vocaset? (of course I will change 'config_parser.py' script for new dataset)
> 
> Sorry for poor English and thank you for your answer.

I have the same question, it seems nobody response in this repo.

**Comment by TimoBolkart** [comment]
True, the audio is input, the mesh data output / target for the training. It is not just about copying the data but you need to bring them into the right data structure, same as the provided data that are loaded in the training script.
